project/SparkBuild.scala

```diff
@@ -458,7 +458,7 @@ object Core {
     resourceGenerators in Compile += Def.task {
       val buildScript = baseDirectory.value + "/../build/spark-build-info"
       val targetDir = baseDirectory.value + "/target/extra-resources/"
-      val command =  buildScript + " " + targetDir + " " + version.value
+      val command =  "bash " + buildScript + " " + targetDir + " " + version.value
```

Comment:
Since you're here, could you change this to use a `Seq` instead of a concatenated string for the command? Concatenating strings doesn't gel very well with spaces.